packages/wrangler/src/type-generation/index.ts

```diff
@@ -511,6 +512,15 @@ export async function generateEnvTypes(
 		}
 	}
 
+	if (configToDTS.pipelines) {
+		for (const pipeline of configToDTS.pipelines) {
+			envTypeStructure.push([
+				constructTypeKey(pipeline.binding),
+				`import("cloudflare:pipelines").Pipeline`,
```

Comment:
Having tested this, this actually needs to be:

```
import("cloudflare:pipelines").Pipeline<
		import("cloudflare:pipelines").PipelineRecord
	>
```

because the `Pipeline` interface has no default for the generic. Is that intentional?